app/controllers/application_controller.rb

```diff
@@ -46,6 +47,11 @@ def create_user_event(event_type, user = current_user)
 
   private
 
+  def disable_caching
+    response.headers['Cache-Control'] = 'no-store'
```

Comment:
just wanted to note a +1 since `no-store` is superior to `no-cache` which is what the Rails `expires_now` method does.